app.js

```diff
@@ -1,12 +1,17 @@
+// Load environment variables from .env file
+require("dotenv").config();
```

Comment:
There's no need to configure the `dotenv` in this file anymore when handling your env configurations from a [separate file](src/configs/env.config.js). Instead, import your [env.config](src/configs/env.config.js) file before importing any other module